mllib/src/main/scala/org/apache/spark/ml/r/AFTSurvivalRegressionWrapper.scala

```diff
@@ -129,7 +129,9 @@ private[r] object AFTSurvivalRegressionWrapper extends MLReadable[AFTSurvivalReg
       val rMetadata = ("class" -> instance.getClass.getName) ~
         ("features" -> instance.features.toImmutableArraySeq)
       val rMetadataJson: String = compact(render(rMetadata))
-      sc.parallelize(Seq(rMetadataJson), 1).saveAsTextFile(rMetadataPath)
+      sparkSession.createDataFrame(
```

Comment:
This is a regression because `SparkSession` is heavier than `SparkContext`.

Reply:
I think it is not a regression, we are using active spark session but not creating a new one.

Reply:
this is what ML scala using for model save and load:

https://github.com/apache/spark/blob/3ae78c4c39a7084a321f2e01b4745cb6c442b7a5/mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala#L493-L523

as a R wrapper, I think it is fine to do it in the same way